src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+
+  gap: $catalog-gap;
+
+  &__card {
+    display: grid;
```

Comment:
here you may just use `flex`, cause you don't need `grid` to place text int the center 😉